### PR TITLE
fix typo in link to values.yaml

### PR DESCRIPTION
**Pull request by shaunc**
doc: defaultClusterIssuer values link was misspelt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ No modules.
 | [utils_deep_merge_yaml.values](https://registry.terraform.io/providers/cloudposse/utils/latest/docs/data-sources/deep_merge_yaml) | data source |
 
 ## Inputs
-
+as
```
Comment (martinhaus) [inline README.md:75]: is this a typo?

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ No modules.
 | <a name="input_argo_project"></a> [argo\_project](#input\_argo\_project) | ArgoCD Application project | `string` | `"default"` | no |
 | <a name="input_argo_sync_policy"></a> [argo\_sync\_policy](#input\_argo\_sync\_policy) | ArgoCD syncPolicy manifest parameter | `map` | `{}` | no |
 | <a name="input_cluster_issuer_enabled"></a> [cluster\_issuer\_enabled](#input\_cluster\_issuer\_enabled) | Variable indicating whether default ClusterIssuer CRD is enabled | `bool` | `false` | no |
-| <a name="input_cluster_issuer_settings"></a> [cluster\_issuer\_settings](#input\_cluster\_issuer\_settings) | Additional settings which will be passed to the Helm chart cluster\_issuer values, see https://github.com/lablabs/terraform-aws-eks-aws-cert-manager/blob/master/helm/defaultClusterIssuer/values.yaml | `map(any)` | `{}` | no |
+| <a name="input_cluster_issuer_settings"></a> [cluster\_issuer\_settings](#input\_cluster\_issuer\_settings) | Additional settings which will be passed to the Helm chart cluster\_issuer values, see https://github.com/lablabs/terraform-aws-eks-cert-manager/blob/master/helm/defaultClusterIssuer/values.yaml | `map(any)` | `{}` | no |
```
Comment (martinhaus) [inline README.md:90]: These are auto-generated from `variables.tf`
https://github.com/shaunc/terraform-aws-eks-cert-manager/blob/patch-1/variables.tf#L102

**Review by martinhaus — CHANGES_REQUESTED** [review]
Can you apply your fix to the variable description instead of the readme file?

File: README.md (hunk 2 above)
Comment (shaunc) [reply README.md:90]: Ah -- right; sorry! I was unfamiliar with the module publication process.... added the fix in variables.tf.

File: README.md (hunk 1 above)
Comment (martinhaus) [reply README.md:75]: @shaunc can you remove this?

File: README.md (hunk 2 above)
Comment (martinhaus) [reply README.md:90]: can you also run terraform docs? it will regenerate the documentation for you

**Comment by dojci** [comment]
Thank you for notice, typo has been fixed 139fa0b0eb94ca9c81997b780f265f46d7ddb9ac

**Pull request closed without merging**